### ISSUE TITLE
Add newline <br /> linebreak support

### ISSUE THREAD

**Issue by andy0x10** [Jan0660/DocFxMarkdownGen#8]
I've noticed that ther is a mismatch between how newline is handled between docfx and docusaurus.

original class comment:
```
<summary>
list of features:
1) something
2) nice
</summary>
```

In docfx HTML (displays correct newlines):
```
list of features:
1) something
2) nice
```

In Docusaurus (displays as though they're all one paragraph):
`list of features: 1) something 2) nice`

I guess when authoring markdown for Docusaurus they expect you to be explicit about your linebreaks.  That's fine, but in API documentation you typically don't need to do this..Perhaps there is an opportunity for DFMG to add "<br />" to the end of newlines to make it display as intended in Docusaurus?

**Comment by Jan0660** [comment]
The problem in this specific case seems to be that the Docusaurus' markdown does not supported ordered lists like `1)`,`2)`, while using `1.`,`2.` does work:

```csharp
    /// <summary>
    /// Also does nothing.
    /// 1) 1
    /// 2) 2
    /// 
    /// but:
    /// 1. 1
    /// 2. 2
    /// </summary>
```

![image](https://github.com/Jan0660/DocFxMarkdownGen/assets/58996212/e3c5ecb2-b362-4560-a9b6-ece26f25a16e)

The problem with forcing a newline on every newline in the xmldoc would be that a sentence split across multiple lines in the xmldoc would then also be split over multiple lines.

I guess some options like forcing a newline always or only when the line ends with a specific character could be implemented.

**Comment by andy0x10** [comment]
True true, but I didn't just mean for lists, I meant for line breaks in general.  Like if my documentation says:

```
This is a statement.
This is another statement.
Here is a separate statement.
```

Let's pretend that in this case the formating is important..

In docfx it renders as written above, but in Docusaurus that will render as:
```
This is a statement. This is another statement. Here is a separate statement.
```

I think with code comments the original intended formating should be preserved...

But now actually as I write this, I'm no longer sure 🤣 ...  Feel free to close this issue, just wanted to explain my thought  process...

**Comment by Jan0660** [comment]
Implemented some options and `<br/>` in v0.4.0 to get what you want.
